README.md

```diff
@@ -1,2 +1,19 @@
 # pull-to-refresh
+Swipe or pull down to reload a page
+
+## Features
+* Pull down to refresh a page
+
```

Comment:
Did you mean adding more to that feature?
Modified  it to: Pull down page until the text 'release to refresh' appears in order to refresh the page